app/cdap/components/DataPrep/Directives/Calculate/index.js

```diff
@@ -44,19 +44,22 @@ export default class Calculate extends Component {
     super(props);
     this.VALID_TYPES = NUMBER_TYPES.concat(['string']);
 
-    const {isDisabled, crossColumn, columns, columnType} = this.parseColumns();
+    const {isDisabled, crossColumn, multiColumn, columns, columnTypes, areColsMixedTypes} = this.parseColumns();
```

Comment:
With this many fields, it's best to split them up on separate lines. So this should look like
```
    const {
      isDisabled,
      crossColumn,
      multiColumn,
      columns,
      columnTypes,
      areColsMixedTypes
    } = this.parseColumns();
```

Reply:
Done!